index.js

```diff
@@ -1,88 +0,0 @@
-const readline = require('readline');
-const rl = readline.createInterface(process.stdin, process.stdout);
-
-function ask(questionText) {
-  return new Promise((resolve, reject) => {
-    rl.question(questionText, resolve);
-  });
-}
-
-initialize();
-
-async function initialize() {
-    let userChoice = await ask("Would you like to play a game where the computer (A) guesses the number, or the human (B) does? ")
-    if (userChoice === "A") {
-        compGame();
-    } else if (userChoice === "B") {
-        humanGame();
-    } else {
-        console.log("Silly human! Please try again.")
-        initialize();
-    }
-};
-
-compGame();
-
-async function compGame() {
-
-  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.");
-
-  let min = 1;
-  let max = 100;
-  let guessComp = (Math.floor(Math.random() * (max - min + 1)) + min); //Random number generator for range 1-100 (Too limiting?)
-
-  let compQuestion = await ask("Is your secret number: " + guessComp + "?" + "\n(Y)es or (N)o: "); { //Why does a typed response of "y" or "n" (lowercase-y or -n) make the game quit? (see lines 52-53) Getting lowercase-y or -n back seems to be the only instance where this occurs.
```

Comment:
the game quits here because when the user gives a response of anything other then "N" you never enter your while loop. the program then skips down to whatever code is immediately after the loop which is the `console.log` and the `process.exit`